framework/include/distributions/Distribution.h

```diff
@@ -0,0 +1,37 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef DISTRIBUTION_H
+#define DISTRIBUTION_H
+
+#include "MooseObject.h"
+#include "DistributionBase.h"
+#include "SetupInterface.h"
+
+class Distribution;
+
+template<>
+InputParameters validParams<Distribution>();
+
+class Distribution :
+  public MooseObject,
+  public SetupInterface,
```

Comment:
I don't think we will need this class at all, since it doesn't add any functionallity beyond what is in DistributionBase. So, I recommend just renaming DistributionBase to Distribution and remove this class.